packages/schematics/angular/utility/find-module.ts

```diff
@@ -25,6 +24,7 @@ export interface ModuleOptions {
   skipImport?: boolean;
   moduleExt?: string;
   routingModuleExt?: string;
+  /** @deprecated This option is no longer required and will be removed in version 9 */
```

Comment:
These are all internal.  The deprecation isn’t necessary here.

Reply:
The thing is that this is used within google so while it’s used internally if we remove it it will break inside google.

Reply:
That would actually be more of a reason to completely remove it since you would then get a compile time error instead of differing runtime behavior; which may not be initially noticed.

Reply:
Removing it is okay too, but @kyliau should probably try a g3sync to see how difficult it will be to keep the sync going.
We haven't synced it in a while IIRC so it might already be hard to do...

Reply:
Removed